src/components/cardContainer/CardContainer.js

```diff
@@ -55,7 +55,7 @@ function hasSelected(card, isSelected = true) {
 }
 
 export default function CardContainer({resetScore, increaseScore}) {
-    const [cardList, setCardList] = useState(cards)
+    const [cardList, setCardList] = useState(() => cards)
```

Comment:
If we were really bothered about the performance (the performance was fine) we could use lazy initialization, to ensure that we only load this data once 

https://kentcdodds.com/blog/use-state-lazy-initialization-and-function-updates/